server/src/entity/Restaurant.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  OneToMany,
+  JoinTable,
+} from 'typeorm';
+import { Menu } from './Menu';
+
+export enum cuisineTypes {
+  AMERICAN = 'American',
+  CHINESE = 'Chinese',
+  JAPANESE = 'Japanese',
+  MEXICAN = 'Mexican',
+  //SEAFOOD = "Seafood",
+  //CAJUN = "Cajun",
+  //KOREAN = "Korean",
+  //MEDITERRANEAN = "Meditterranean",
+  //CUBAN = "Cuban",
+  //MIDDLE_EASTERN = "Middle Eastern",
+  //SUSHI_BAR = "Sushi Bar",
+  //AMERICAN = "American",
+  //FRENCH = "French",
+  //DELIS = "Delis",
+  //SANDWICH = "Sandwich",
+  //BAGEL = "Bagel",
+  //STEAKHOUSE = "Steakhouse",
+  //THAI = "Thai",
+  //UNKNOWN = "Unknown"
+}
+
+@Entity()
+export class Restaurant extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  public id: string;
+
+  @Column({
+    length: 50,
+  })
+  public name: string;
+
+  @Column({
+    unique: true,
+    nullable: true,
+  })
+  public email: string;
+
+  @Column({
+    unique: true,
+  })
+  public location: string;
```

Comment:
Anyway we can do coordinates?

Reply:
added address attribute and location attribute which contains longitude and latitude